src/libertem/io/dataset/k2is.py

```diff
@@ -641,7 +642,7 @@ def _get_syncer(self, do_sync=True):
 
     def _get_fileset(self, with_sync=True):
         sig_shape = (SECTOR_SIZE[0], NUM_SECTORS * SECTOR_SIZE[1])
-        num_frames = int(np.prod(self._get_scansize()))
+        num_frames = int(np.prod(self._scan_size))
```

Comment:
@sk1p This was probably a merge artifact? What is correct here?

Reply:
Change looks good, as `_get_scansize` accesses the gtg file, which means `_get_fileset` could only be used on worker nodes.

Reply:
Thx! :-)